staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/kcp_naming_controller_test.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2022 The kcp Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
+
+import (
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	apiextensionshelpers "k8s.io/apiextensions-apiserver/pkg/apihelpers"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	listers "k8s.io/apiextensions-apiserver/pkg/client/listers/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+	"testing"
+	"time"
+)
+
+func newBoundCRD(resource, group string) *crdBuilder {
+	return &crdBuilder{
+		curr: apiextensionsv1.CustomResourceDefinition{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: uuid.New().String(),
+				Annotations: map[string]string{
+					"apis.kcp.dev/bound-crd": "",
+				},
+			},
+			Spec: apiextensionsv1.CustomResourceDefinitionSpec{
+				Group: group,
+				Names: apiextensionsv1.CustomResourceDefinitionNames{
+					Plural: resource,
+				},
+			},
+		},
+	}
+}
+
+func TestSync_KCP_BoundCRDsDoNotConflict(t *testing.T) {
```

Comment:
is kcp_* our new naming convention? Am fine with it, we should just declare it as convention "publicly".

Reply:
I don't know that we have anything formal established yet, but we need to.